api/types/types.go

```diff
@@ -51,7 +51,7 @@ const (
 	TransportServiceAccountName   string = "skupper-router"
 	TransportRoleName             string = "skupper-router"
 	TransportRoleBindingName      string = "skupper-router"
-	TransportEnvConfig            string = "QDROUTERD_CONF"
+	TransportEnvConfig            string = "SKROUTERD_CONF"
```

Comment:
Prior to changing this, we need to change the environment variable name in `skupper-router`
as it is still being referred there with the old name.